Userland/Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -72,6 +73,28 @@ void ArrayPrototype::initialize(GlobalObject& global_object)
     // evaluates to true
     // 23.1.3.33 Array.prototype [ @@iterator ] ( ), https://tc39.es/ecma262/#sec-array.prototype-@@iterator
     define_property(vm.well_known_symbol_iterator(), get(vm.names.values), attr);
+
+    // 23.1.3.34 Array.prototype [ @@unscopables ], https://tc39.es/ecma262/#sec-array.prototype-@@unscopables
+    {
+        Object* unscopable_list = create_empty(global_object);
+        unscopable_list->set_prototype(nullptr);
+        for (auto name : {
+                 vm.names.copyWithin,
+                 vm.names.entries,
+                 vm.names.fill,
+                 vm.names.find,
+                 vm.names.findIndex,
+                 vm.names.flat,
+                 vm.names.flatMap,
+                 vm.names.includes,
+                 vm.names.keys,
+                 vm.names.values,
+             }) {
+            unscopable_list->put(name, Value(true));
+        }
+
+        define_property(vm.well_known_symbol_unscopables(), unscopable_list->value_of(), Attribute::Configurable);
+    }
```

Comment:
Why is this wrapped by a scope like this? Also, this would be both shorter and IMO more readable without the loop (just like the spec does it).